app/views/entities/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="flex flex-col items-center mb-6 pb-6 bg-slate-50">
+<%= render "header" %>
+  <div class="flex w-80 m-2">
+    <%= image_tag @group.icon, alt: 'icon', class: "w-28" %>
+    <p class="ml-10 text-xl text-gray-500 font-medium"><%= @group.name %></p>
+  </div>
+
+  <div class="flex justify-between w-80 mt-4 text-lg">
+  <p class="text-gray-500">Total Payment</p>
+  <p class="font-semibold">$<%= @total_amount %></p>
+  </div>
+  <hr class="mt-3">
+  <% @entities.each_with_index do |entity, index| %>
+  <div class="mt-6 w-80 text-gray-500">
+    <p>Transaction N<%= index+8567%> <%= entity.name %> </p>
+    <div class="flex mt-4 gap-2">
+      <%= image_tag ('all/calender-icon.png'), class: "w-3" %>
+      <p class="text-xs"><%= entity.created_at.strftime('%d %h %Y') %></p>
+    </div>
+
+    <div class="flex mt-4 justify-between">
+      <p>Payment</p>
+      <p class="text-slate-900 font-semibold">$<%= entity.amount %></p>
+  </div>
+  </div>
+  <hr class="mt-3">
+  <% end %>
+
+  <%= link_to 'Add new transaction', new_group_entity_path, class: "bg-green-400 w-96 text-center p-2 rounded text-slate-50 mt-6" %>
```

Comment:
- [ ] Kindly ensure the add new transaction button sticks to the bottom of the page.